cmd/cerbosctl/inspect/policies.go

```diff
@@ -28,13 +28,20 @@ cerbosctl inspect policies`
 	separator = ","
 )
 
+//nolint:govet
 type PoliciesCmd struct {
 	flagset.Filters
 	flagset.Format
+
+	PolicyIDs []string `arg:"" name:"id" optional:"" help:"list of policy ids"`
```

Comment:
I was confused by this until I realised that `inspect` command has its own `flagset.Filters` which is nearly identical to the `get` command filter flags except for `name` and `version`. That's fair enough. However, there are two inconsistencies in here now.
- One `flagset.Filters` definition includes `PolicyIDs` while the other doesn't. That's not ideal in terms of code consistency.
- The validation logic for flags is now inconsistent: the `get` command doesn't allow using `--include-disabled` if any `policyIDs` are defined but, the `inspect` command does. That's a user-facing inconsistency. 

I think that that validation check for `--include-disabled` + `policyIDs` in `get` is not necessary because `include-disabled` is a harmless no-op in that case anyway. It should be removed so that we don't throw unnecessary errors at users. I'd also try to keep the `flagset.Filters` definitions as similar as possible in the code.